exchanges/gateio/gateio_wrapper.go

```diff
@@ -1926,8 +1928,7 @@ func (g *Gateio) GetAvailableTransferChains(ctx context.Context, cryptocurrency
 	if err != nil {
 		return nil, err
 	}
-
-	availableChains := make([]string, len(chains))
+	availableChains := make([]string, 0, len(chains))
 	for x := range chains {
 		if chains[x].IsDisabled == 0 {
 			availableChains[x] = chains[x].Chain
```

Comment:
As suggested in this request [HERE](https://github.com/thrasher-corp/gocryptotrader/pull/1058#discussion_r1022307456)

Please append, this panics.